src/nedap/speced/def/impl/let_impl.cljc

```diff
@@ -16,9 +16,10 @@
   (let [bindings (->> bindings
                       (partition 2)
                       (map (fn [[left right]]
-                             (let [analysis-result (process-name-and-tails {:tail (list [left])
-                                                                            :name nil
-                                                                            :clj? clj?})
+                             (let [analysis-result (process-name-and-tails {:tail      (list [left])
+                                                                            :name      nil
+                                                                            :checking? false
```

Comment:
change to `true`, stop parsing prepost, discard `defn add-prepost`, ensure sensible `checking` clauses are emitted for `let`